core/src/main/java/org/mskcc/cbio/portal/util/CancerStudyPermissionEvaluator.java

```diff
@@ -175,7 +175,12 @@ private boolean hasPermission(CancerStudy cancerStudy, Authentication authentica
         } catch (DaoException e) {
             groups = cancerStudy.getGroups();
         }
-        if (!Collections.disjoint(groups, grantedAuthorities)) {
+        // to upperCase to make the comparison with grantedAuthorities case insensitive:
```

Comment:
I wanted to avoid changing  cancerStudy.getGroups() as it could be used somewhere else.